### DISCUSSION TITLE
Can not debug hyper hello example

### DISCUSSION THREAD

**Discussion by sec-researcher** [SeaQL/FireDBG.for.Rust#25] [Q&A] [answered]
Hi guys

I want to debug hyper 1.0.1 hello example, the firedbg run and return this warning: 
warning: This version of LLDB has no plugin for the language "assembler". Inspection of frame variables will be limited.

Also after sending http request to the port the debug is just able to track first function after main not any more.
![Screenshot from 2023-12-22 18-19-38](https://github.com/SeaQL/FireDBG.for.Rust/assets/7353420/51f217c6-bbe3-456e-b693-825b6a689cec)

**Comment by tyt2y3** [comment, marked as answer]
Thank you for trying out FireDBG. Async rust support is still work in progress.
